client/a8c-for-agencies/sections/marketplace/hosting-overview-v3/hosting-content/enterprise-agency-hosting/index.tsx

```diff
@@ -1,14 +1,17 @@
-import { VIPLogo, SalesforceLogo, SlackLogo, CNNLogo } from '@automattic/components';
+import { VIPLogo } from '@automattic/components';
 import { Button } from '@wordpress/components';
 import { Icon, external } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
 import { BackgroundType11 } from 'calypso/a8c-for-agencies/components/page-section/backgrounds';
 import SimpleList from 'calypso/a8c-for-agencies/components/simple-list';
 import ProfileAvatar1 from 'calypso/assets/images/a8c-for-agencies/hosting/enterprise-testimonial-1.png';
 import ProfileAvatar2 from 'calypso/assets/images/a8c-for-agencies/hosting/enterprise-testimonial-2.png';
+import CNNLogo from 'calypso/assets/images/logos/cnn.svg';
```

Comment:
Seems like we have a CNN logo here:

https://github.com/Automattic/wp-calypso/blob/3a11252a2090b9f5113aaf84475becb450361495/client/assets/images/start-with/CNN.svg

Yes, it's white, but we should be able to make it black by specifying `fill: #000` to its `path`.

So, could we possibly use the same svg file in all places?